README.md

```diff
@@ -1,80 +1,112 @@
+| `deploy-status` | true        | The status of the deployment.  <br/><br/>Possible Values: *success, failure, cancelled, skipped*                                                                                                                                 |
+| `timezone`      | false       | IANA time zone name (e.g. America/Denver) to display dates in.  If time zone is not provided, dates will be shown in UTC                                                                                                         |
+
 
-## Outputs
-| Output     | Description           |
-| ---------- | --------------------- |
-| `output-1` | Description goes here |
 
 ## Example
 
 ```yml
-# TODO: Fill in the correct usage
+name: Manually Deploy to QA
+on:
+  workflow_dispatch:
+    inputs:
+      branchTagOrSha:
+        description: 'The branch, tag or sha to deploy '
+        required: false
+      refType:
+        description: 'The ref type (branch, tag, sha)'
+        required: false  
+
 jobs:
-  job1:
+  environment: 'QA'
+  deploy-different-ways:
     runs-on: [self-hosted, ubuntu-20.04]
     steps:
       - uses: actions/checkout@v2
 
-      - name: Add Step Here
-        uses: im-open/this-repo@v1
+      - id: deploy-to-qa
+        continue-on-error: true  #Setting to true so the deployment board can be updated, even if this fails
+        run: |
+          ./deploy-to-qa.sh
+        
+        # Defaults to using github-actions for the login, regex matching to determine the ref-type and times shown in UTC
+      - name: Update deployment board with Defaults
+        continue-on-error: true                             # Setting to true so the job doesn't fail if updating the board fails.
+        uses: im-open/update-deployment-board@v1.0.0
         with:
-          input-1: 'abc'
-          input-2: '123
+          github-token: ${{ secrets.GITHUB_TOKEN}}          # If a different token is used, update github-login with the corresponding account
+          environment: 'Dev'
+          board-number: 1
+          ref: ${{ github.event.inputs.branchTagOrSha }}
+          deploy-status: ${{ steps.deploy-to-qa.outcome }}  # outcome is the result of the step before continue-on-error is applied
+      
+      - name: Update deployment board with all values provided
+        continue-on-error: true                             # Setting to true so the job doesn't fail if updating the board fails.
+        uses: im-open/update-deployment-board@v1.0.0
+        with:
+          github-token: ${{ secrets.BOT_TOKEN}}             # If a different token is used, update github-login with the corresponding account
+          github-login: 'my-bot'
+          environment: 'QA'
+          board-number: 2
+          ref: 'feature-branch-16'
+          ref-type: 'branch' 
+          deploy-status: ${{ steps.deploy-to-qa.outcome }}  # outcome is the result of the step before continue-on-error is applied
+          timezone: 'america/denver'
 ```
```

Comment:
I think i'd add to the example a "fail the build" step that returns the error from the deploy step if there was one.
Something like: [this](https://github.com/bc-swat/github-services-infrastructure/blob/31b9e9e46b9d80d44036b494cf39d54a7bb74ca9/.github/workflows/tf-actions-runners-auto-plan-and-comment-on-prs.yml#L91)

Reply:
I meant to do that and forgot.